applications/sink/rabbit-sink/src/test/java/org/springframework/cloud/stream/app/sink/rabbit/HeadersMappedLastTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2016-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.app.sink.rabbit;
+
+import org.junit.jupiter.api.Test;
+
+import org.springframework.amqp.core.Message;
+import org.springframework.integration.support.MessageBuilder;
+import org.springframework.test.context.TestPropertySource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class HeadersMappedLastTest {
+
+	@TestPropertySource(properties = { "rabbit.routingKey=scsapp-testq",
+			"rabbit.headersMappedLast=true" })
```

Comment:
We probably don't need this property here since it is `true` by default now. So, we just would test it like "and what it is a default behavior?"